### PR TITLE
chore: update ape pin to v0.4.0

### PR DESCRIPTION
**Pull request by sabotagebeats**
### What I did
updated pin to ape v0.4.0

### How I did it
update setup.py

### How to verify it
github checks

### Checklist

- [ ] Passes all linting checks (pre-commit and CI jobs)
- [ ] New test cases have been added and are passing
- [ ] Documentation has been updated
- [ ] PR title follows [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) standard (will be automatically included in the changelog)

### CODE REVIEW

**Comment by NotPeopling2day** [comment]
Looks good but would you add the additional changes to match https://github.com/ApeWorX/project-template/pull/22

Thanks!

**Review by johnson2427 — APPROVED** [review]
LGTM

File: TokenProject/contracts/Token.vy [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+# ERC20 Token Metadata
+NAME: constant(String[20]) = "TokenTest"
+SYMBOL: constant(String[5]) = "TKN"
+DECIMALS: constant(uint8) = 18
+
+# ERC20 State Variables
+totalSupply: public(uint256)
+balanceOf: public(HashMap[address, uint256])
+allowance: public(HashMap[address, HashMap[address, uint256]])
+
```
Comment (NotPeopling2day) [inline TokenProject/contracts/Token.vy:16]: @unparalleled-js I think you added some stuff you didn't intend to be in the template?

Comment (unparalleled-js) [reply TokenProject/contracts/Token.vy]: fixed! It was related to an issue of running the tests from outside the tests directory so I made a fixture that set the currdir there as well so they won't show up again.

**Review by NotPeopling2day — APPROVED** [review]

**Pull request merged**